adafruit_trellism4.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, pin, *, width, height, rotation=0):
     def __setitem__(self, index, value):
         if not isinstance(index, tuple) or len(index) != 2:
             raise IndexError("Index must be tuple")
+        if index[0] > 7 or index[1] > 7:
```

Comment:
I think could be a tighter check:
```
if index[0] >= self.width or index[1] > self.height:
    raise IndexError(...)
```

`index[0]` is always the width, right?

Reply:
also check for the indexes < 0

Reply:
Yes, and it looks like this works. Adding now!